website/templates/includes/footer.html

```diff
@@ -2,7 +2,7 @@
 {% load i18n %}
 {% load custom_tags %}
 {% comment %} 'img/logo_white.png' {% endcomment %}
-<footer class="bg-black text-white pt-12 pb-8 px-12 bottom-0 relative fixed t-0">
+<footer class="bg-black text-white pt-12 pb-8 px-12 relative ">
```

Comment:
what does this do? is it related to the add a new issue page?

Reply:
@AtmegaBuzz what does this do?  Is it needed for the add a new issue page?  Please revert it if not - would like a minimal amount of changes files